client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1768,7 +1768,7 @@
                     if (angular.isDefined(item)) {
                         item._datelinedate = '';
 
-                        if (angular.isDefined(item.dateline.located) && !_.isNull(item.dateline.located)) {
+                        if (item.dateline && item.dateline.located != null) {
```

Comment:
We should check for `undefined of item.dateline.located`. Sometimes it's undefined. Also it should be `item.dateline.located !== null`